src/renderer/src/Pages/SavedEnvironments/SavedEnvironments.tsx

```diff
@@ -0,0 +1,87 @@
+import Loader from "@renderer/Layout/Loader";
+import { getSavedEnvironments } from "@renderer/lib/environments";
+import { environment } from "@renderer/lib/interfaces";
+import { motion } from "framer-motion";
+import { useEffect, useState } from "react";
+import toast from "react-hot-toast";
+import Environment from "../Profile/Components/Environment";
+import { TbLayoutGrid, TbLayoutList } from "react-icons/tb";
+import { userAtom } from "@renderer/lib/atoms";
+import { useAtomValue } from "jotai";
+import { useNavigate } from "react-router-dom";
+
+export default function SavedEnvieonments(): JSX.Element {
```

Comment:
```suggestion
export default function SavedEnvironments(): JSX.Element {
```

Remember to change all references too